pkg/logout/logout.go

```diff
@@ -30,7 +31,7 @@ func Logout(config *config.Config) error {
 		fmt.Printf("Credentials have been cleared for %s.\n", profileName)
 	}
 
-	return nil
+	return config.UseProfile(hookdeck.DefaultProfileName)
```

Comment:
Currently when logging out, I'm "resetting" the profile config back to default. This will update or create a new local hookdeck.toml file. This is definitely not the desirable behavior, especially for existing users who don't have the local hookdeck.toml file yet.

The behavior of this is kinda related to the "global vs local" conversation in the description so I'm calling this out for now and we'll make changes for this later once we have a better idea what to do.